packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -62,6 +62,14 @@ $ms-Persona-presenceSizeMd: 12px;
 $ms-Persona-presenceSizeLg: 20px;
 $ms-Persona-presenceSizeXl: 28px;
 
+:global(.ms-PickerPersona-container.is-selected) {
```

Comment:
Do we have to use `:global` to target this? We can add in new CSS modules classes if they're missing.